pkgs/os-specific/darwin/apple-sdk-11.0/libcompression.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenvNoCC, buildPackages, MacOSX-SDK }:
+
+let self = stdenvNoCC.mkDerivation {
+  pname = "libcompression";
```

Comment:
Oh, I think this is probably a better solution than https://github.com/NixOS/nixpkgs/pull/230390/files#diff-3e4b9f931538d331c1402357df553700a107e167d0a6197a53e6c85e3e42aa03R252